pkg/cmd/server/origin/controller.go

```diff
@@ -148,5 +193,34 @@ func (c *MasterConfig) NewOpenshiftControllerInitializers() (map[string]controll
 	}
 	ret["openshift.io/service-serving-cert"] = serviceServingCert.RunController
 
+	sdnController := controller.SDNControllerConfig{
+		NetworkConfig: c.Options.NetworkConfig,
+	}
+	ret["openshift.io/sdn"] = sdnController.RunController
+
+	ret["openshift.io/resource-quota-manager"] = controller.RunResourceQuotaManager
```

Comment:
Run this under its kube name for now and open an issue to split upstream from downstream.  Given shared informers, the watch cost in negligible if the update conflicts can be borne.